File: app/src/main/java/com/krystianwsul/checkme/domainmodel/RemoteProjectRelevance.java
package com.krystianwsul.checkme.domainmodel;

import android.support.annotation.NonNull;

import com.krystianwsul.checkme.firebase.RemoteProject;

public class RemoteProjectRelevance {
    @NonNull
    private final RemoteProject mRemoteProject;

    private boolean mRelevant = false;

    RemoteProjectRelevance(@NonNull RemoteProject remoteProject) {
        mRemoteProject = remoteProject;
    }

    public void setRelevant() {
        mRelevant = true;
    }

    boolean getRelevant() {
        return mRelevant;
    }

    @NonNull
    RemoteProject getRemoteProject() {
        return mRemoteProject;
    }
}

File: app/src/main/java/com/krystianwsul/checkme/domainmodel/RemoteProjectRelevance.kt
package com.krystianwsul.checkme.domainmodel

import com.krystianwsul.checkme.firebase.RemoteProject

class RemoteProjectRelevance(val remoteProject: RemoteProject) {

    var relevant = false
        private set

    fun setRelevant() {
        relevant = true
    }
}
